artemis/build.gradle

```diff
@@ -38,6 +38,8 @@ dependencies {
 
   testImplementation 'com.squareup.okhttp3:okhttp'
   testImplementation 'org.awaitility:awaitility'
+  testImplementation 'org.mockito:mockito-core'
+  testImplementation 'org.mockito:mockito-junit-jupiter'
```

Comment:
We've generally just used the static mockito methods so not needed the `mockito-junit-jupiter` dependency.  Would be good to remain consistent.

Reply:
Interesting, alright static it is 👍